test/CMakeLists.txt

```diff
@@ -15,6 +15,18 @@
 set (ranlux_24_48_test.pass_timeout_debug "900") # 15min
 set (ranlux_24_48_test.pass_timeout_release "720") # 12min
 
+# If TBB headers are available, libstdc++ enables TBB backend; some TBB symbols become undefined unless it is linked.
+# The undefined symbol error arises only with GCC compiler.
+if (CMAKE_CXX_COMPILER_ID STREQUAL GNU AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER_EQUAL 9)
+    if (ONEDPL_BACKEND MATCHES "^(serial|omp|dpcpp_only)$")
+        find_package(TBB)
```

Comment:
If I am understanding this correctly, it doesn't matter what version of TBB is linked, is that correct? (tbb vs oneTBB)

Reply:
Thanks for highlighting it. 

I think, a more robust approach would be to search TBB 2020 (or older) for libstdc++ 9/10 and oneTBB 2021 for newer libstdc++ versions. 

However, in order not to overcomplicate the logic, let's limit the workaround to libstdc++ 11 and newer and search oneTBB, because this workaround looks unnecessary for libstdc++ 9/10:
 - As far as I understand, oneDPL supports only oneTBB
 - oneDPL will just turn off TBB in libstdc++ 9/10 as noted by Adam in https://github.com/oneapi-src/oneDPL/pull/553#discussion_r1382161802.